packages/clerk-js/src/ui/elements/Card/CardContent.tsx

```diff
@@ -31,16 +28,16 @@ export const CardContent = React.forwardRef<HTMLDivElement, CardContentProps>((p
           boxShadow: t.shadows.$cardContentShadow,
           borderRadius: t.radii.$lg,
           position: 'relative',
-          padding: t.space.$8,
-          gap: t.space.$8,
+          padding: `${t.space.$8} ${t.space.$10}`,
           justifyContent: 'center',
           alignContent: 'center',
         }),
         sx,
       ]}
+      gap={gap}
```

Comment:
🙃 This could be `gap={8}` without decontructing it from the rest of the props.

Reply:
Fixing!